cmake/modules/Logging.cmake

```diff
@@ -35,21 +26,30 @@ if("${USE_LIBBACKTRACE}" STREQUAL "AUTO")
   message(STATUS "Autoset: USE_LIBBACKTRACE=" ${USE_LIBBACKTRACE} " in " ${CMAKE_SYSTEM_NAME})
 endif()
 
+if (USE_LIBBACKTRACE)
+    set(current_dir ${CMAKE_CURRENT_LIST_DIR})
+endif ()
 
-if(USE_LIBBACKTRACE)
-  message(STATUS "Building with libbacktrace...")
-  include(cmake/libs/Libbacktrace.cmake)
-  target_link_libraries(tvm PRIVATE libbacktrace)
-  target_link_libraries(tvm_runtime PRIVATE libbacktrace)
-  add_dependencies(tvm_runtime_objs libbacktrace)
```

Comment:
This line needs to be kept because `tvm_runtime_objs` depends on header files generated by libbacktrace.